cognite/client/_api_client.py

```diff
@@ -412,6 +412,20 @@ def get_partition(partition):
             raise tasks_summary.exceptions[0]
         return cls._load(tasks_summary.joined_results(), cognite_client=self._cognite_client)
 
+    def _aggregate(
```

Comment:
This should be unit tested in test_api_client.py

Reply:
added